gammapy/image/catalog.py

```diff
@@ -2,18 +2,138 @@
 """Make an image from a source catalog, or simulated catalog, e.g 1FHL 2FGL etc
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from collections import OrderedDict
+
 import numpy as np
 from astropy.wcs import WCS
 from astropy.units import Quantity
+from astropy import units as u
 from astropy.table import Table
 
+from .models import Delta2D
+from .core import SkyImage
+from .lists import SkyImageList
+
 
 __all__ = [
+    'CatalogImageEstimator',
     'catalog_image',
     'catalog_table',
 ]
 
 
+BBOX_DELTA2D_PIX = 5
+
+class CatalogImageEstimator(object):
+    """
+    Compute model image for given energy band from a catalog.
```

Comment:
FYI: I've taken a decision to follow the `pycodestyle` default settings and have the summary line always on directly after the `"""`. Most of Gammapy is made consistent this way, I'll do the rest next week before the Gammapy 0.7 release. Please follow that style for new docstrings.  There's a few other minor nitpicks that `pydocstyle --style=numpy gammapy/image/catalog.py` will point out.

Note that like code style, docstring style is rather arbitrary, and the main point of having a style is consistency and not having to think / argue about it. My suggestion is that we just the default style of `pydocstyle --style=numpy` unless several core developers feel it's bad in some way and we need to modify it.